ClientApp/viking-vault/src/Router.tsx

```diff
@@ -63,14 +64,21 @@ class Router extends React.Component<any, IState> {
             setTimeout(() => {
                 if(this.state.isAdmin)
                     return <AdminPage/>;
-                else return <Redirect to = "/login" />
-            },200);
+                else 
+                    return <Redirect to = "/login" />;
+            },500);
+        }
+        else
+        {
```

Comment:
In the JS community, it's a sort of an unwritten rule/convention to write `{` on the same line